R/stan_betareg.R

```diff
@@ -43,33 +43,38 @@
 #' @template args-QR
 #' @template args-sparse
 #' 
-#' @param link Character specification of the link function used on the
-#'   linear predictor for mu (specified through \code{x}).
-#'   Currently, "logit", "probit", "cloglog", "cauchit", "log", "loglog"
-#'   are supported.
-#' @param link.phi Character specification of the link function used on the
-#'   linear predictor for phi (specified through \code{z}).
-#'   Currently, "identity", "log", and "sqrt" are supported. Since the
-#'   "sqrt" link function is known to be unstable, it is advisable to
-#'   specify a different link function (or to model phi as a parameter instead
-#'   of a linear predictor).
-#' @param z Regressor matrix for phi. Defaults to an intercept
-#'   only.
-#' @param prior_z See \code{prior}.
-#' @param prior_intercept_z See \code{prior_intercept}.
-#' @param prior_phi See \code{prior_dispersion}.
+#' @param link Character specification of the link function used in the model 
+#'   for mu (specified through \code{x}). Currently, "logit", "probit",
+#'   "cloglog", "cauchit", "log", "loglog" are supported.
+#' @param link.phi If applicable, character specification of the link function 
+#'   used in the model for \code{phi} (specified through \code{z}). Currently, 
+#'   "identity", "log", and "sqrt" are supported. Since the "sqrt" link function
+#'   is known to be unstable, it is advisable to specify a different link 
+#'   function (or to model \code{phi} as a scalar parameter instead of via a
+#'   linear predictor).
```

Comment:
@imadmali I tweaked this but can you also add something here to clarify the part in parentheses at the end? That is, how does one not model phi as a linear predictor? I assume by not specifying a `z` part of the formula, but it would be good to make that explicit.